docs/api/notification.md

```diff
@@ -85,6 +85,8 @@ Emitted when the notification is closed by manual intervention from the user.
 This event is not guaranteed to be emitted in all cases where the notification
 is closed.
 
+On Windows, the `close` event is emitted when the notification leaves the screen either by programmatic dismissal via `notification.close()`, the user closing the notification, or via system timeout. If the notification remains in the Action Center afterwards, a further call to `notification.close()` will remove the notification from the action center, but the `close` event will not be emitted again.
```

Comment:
The "either" should be removed since there are 3 cases.

Also the "afterwards" is confusing, it sounds like that the notification could still remain in Action Center after calling `close()`.